client/my-sites/site-settings/form-general.jsx

```diff
@@ -91,40 +92,68 @@ export class SiteSettingsFormGeneral extends Component {
-						<FormLabel htmlFor="blogdescription">{ translate( 'Site tagline' ) }</FormLabel>
-						<FormInput
-							name="blogdescription"
-							id="blogdescription"
-							data-tip-target="site-tagline-input"
-							value={ fields.blogdescription || '' }
-							onChange={ onChangeField( 'blogdescription' ) }
-							disabled={ isRequestingSettings }
-							onClick={ eventTracker( 'Clicked Site Tagline Field' ) }
-							onKeyPress={ uniqueEventTracker( 'Typed in Site Tagline Field' ) }
+			<>
+				<div className="site-settings__site-options">
+					<div className="site-settings__site-title-tagline">
+						<FormFieldset>
+							<FormLabel htmlFor="blogname">{ translate( 'Site title' ) }</FormLabel>
+							<FormInput
+								name="blogname"
+								id="blogname"
+								data-tip-target="site-title-input"
+								value={ fields.blogname || '' }
+								onChange={ onChangeField( 'blogname' ) }
+								disabled={ isRequestingSettings }
+								onClick={ eventTracker( 'Clicked Site Title Field' ) }
+								onKeyPress={ uniqueEventTracker( 'Typed in Site Title Field' ) }
+							/>
+						</FormFieldset>
+						<FormFieldset>
+							<FormLabel htmlFor="blogdescription">{ translate( 'Site tagline' ) }</FormLabel>
+							<FormInput
+								name="blogdescription"
+								id="blogdescription"
+								data-tip-target="site-tagline-input"
+								value={ fields.blogdescription || '' }
+								onChange={ onChangeField( 'blogdescription' ) }
+								disabled={ isRequestingSettings }
+								onClick={ eventTracker( 'Clicked Site Tagline Field' ) }
+								onKeyPress={ uniqueEventTracker( 'Typed in Site Tagline Field' ) }
+							/>
+							<FormSettingExplanation>
+								{ translate( 'In a few words, explain what this site is about.' ) }
+							</FormSettingExplanation>
+						</FormFieldset>
+					</div>
+					<SiteIconSetting />
+				</div>
+				<div className="site-settings__fiverr-logo-maker-cta">
+					<div className="site-settings__fiverr-logo-icon">
+						<img
+							className="site-settings__fiverr-logo-cta"
+							src={ fiverrLogo }
+							alt="fiverr small logo"
 						/>
-						<FormSettingExplanation>
-							{ translate( 'In a few words, explain what this site is about.' ) }
-						</FormSettingExplanation>
-					</FormFieldset>
+					</div>
+					<div className="site-settings__fiverr-logo-maker-cta-text">
+						<div className="site-settings__fiverr-logo-maker-cta-text-title">
+							{ translate( 'Make an incredible logo in minutes' ) }
+						</div>
+						<div className="site-settings__fiverr-logo-maker-cta-text-subhead">
+							{ translate( 'Pre-designed by top talent. Just add your touch.' ) }
+						</div>
+					</div>
+					<div className="site-settings__fiver-cta-button">
+						<Button
+							href={ 'https://wp.me/logo-maker/?utm_campaign=general_settings' }
+							onClick={ this.trackFiverrLogoMakerClick }
+						>
+							<Gridicon icon="external" />
+							{ translate( ' Try Fiverr Logo Maker' ) }
```

Comment:
Is the leading whitespace necessary? If so, it should not be part of the translatable string, an alternative would be:
```suggestion
							{ ' ' }{ translate( '=Try Fiverr Logo Maker' ) }
```